  Generating xr_generated_core_validation.hpp using C:/Program Files (x86)/Microsoft Visual Studio/Shared/Python39_64/python.exe on validation_layer_generator.py
  C:/Program Files (x86)/Microsoft Visual Studio/Shared/Python39_64/python.exe: can't open file 'C:\Users\Chengxiang Li\Desktop\OpenXR-SDK-Source-release-1.1.37\src\scripts\src_genxr.py': [Errno 2] No such file or directory
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(249,5): error MSB8066: Custom build for 'D:\HelloWorldTest\CMakeFiles\f5828221c7736d1b2a2fc8eb1840436b\xr_generated_core_validation.hpp.rule;D:\HelloWorldTest\CMakeFiles\f5828221c7736d1b2a2fc8eb1840436b\xr_generated_core_validation.cpp.rule;D:\HelloWorldTest\CMakeFiles\f5828221c7736d1b2a2fc8eb1840436b\XrApiLayer_core_validation.json.rule;C:\Users\Chengxiang Li\Desktop\OpenXR-SDK-Source-release-1.1.37\src\api_layers\CMakeLists.txt' exited with code 2.
